src/components/carousel/panels/subcomponents/LeftPanel.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+export const DetailsPanel = styled.div`
+  flex: ${props => (props.compact ? "1 0 50%" : "3")};
+  height: 100%;
+  width: 100%;
+  position: relative;
+`;
+
+export const DetailWrapper = styled.div`
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  padding: 0px 3rem;
+  flex: 2;
+  @media screen and (min-width: 992px) {
+    flex: 4;
+  }
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: ${props => (props.dataset ? "flex-end" : "center")};
+`;
+
+export const Headline = styled.h1`
+  font-size: ${props => (props.compact ? "1.3rem" : "1rem")};
```

Comment:
this isn't a necessary change, but i want to raise up an alternative syntax for this pattern that is repeated in lots of places. this is equivalent:

```
export const Headline = styled.div(({ compact }) => `
  font-size: ${ compact ? "1.3rem" : "1rem" };
  // rest of styles 
`)```